dts/bindings/mipi-dbi/mipi-dbi-device.yaml

```diff
@@ -34,3 +34,26 @@ properties:
       - 6
       - 7
       - 8
+
+  te-mode:
+    type: int
+    default: 0
+    description: |
+      MIPI DBI tearing enable signal mode. Use the macros, not the actual enum
+      value. Defaults to disabled.
+      Here is the concordance list (see dt-bindings/mipi_dbi/mipi_dbi.h)
+        0     MIPI_DBI_TE_NO_EDGE
+        1     MIPI_DBI_TE_RISING_EDGE
+        2     MIPI_DBI_TE_FALLING_EDGE
+    enum:
+      - 0
+      - 1
+      - 2
```

Comment:
Use a string property together with `DT_INST_STRING_UPPER_TOKEN` to make it more explicit and clear.

Reply:
Thanks for this idea- updated in latest push